src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Getting Started Page section items</description>
+    <fields>
+        <fullName>Description_Label__c</fullName>
+        <description>The custom label api name used in the description on a section item on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The custom label api name for the Description Label</inlineHelpText>
+        <label>Description Label</label>
+        <length>255</length>
+        <required>true</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Extra_Info_Label__c</fullName>
+        <description>The custom label api name used in the extra info on a section item on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Label which will be rendered at the upper right corner with extra info(video length, pdf, etc)</inlineHelpText>
+        <label>Extra Info Label</label>
+        <length>255</length>
+        <required>false</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>GS_Admin_Checklist_Section__c</fullName>
+        <description>The section this belongs to on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The GettingStarted Section this item is part of</inlineHelpText>
+        <label>GS Admin Checklist Section</label>
+        <referenceTo>GS_Admin_Checklist_Section__mdt</referenceTo>
+        <relationshipLabel>GS Admin Checklist Items</relationshipLabel>
+        <relationshipName>GS_Admin_Checklist_Items</relationshipName>
+        <required>true</required>
+        <type>MetadataRelationship</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Has_Link__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>Does the item have a link to display</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Does the item have a link to display</inlineHelpText>
```

Comment:
Please change this and the <description> to "Indicates if the item has a link to display."